van-gogh-second-page/maria/index-Van-Gogh-for-Children.css

```diff
@@ -0,0 +1,220 @@
+
+* {
+    margin: 0px;
+    padding: 0px;
+}
+
+input, button, submit { border:none; background: none; } 
+
+a {
+    background-color: none;
+    text-decoration: none;
+    color: inherit;
+}
+
+/* Styles */
+body {
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
```

Comment:
This rule relies on Verdana .... fonts stored in a computer which might not be the case for some devices. But we could still do it if it's not crucial to have a specific font here, sans-serif will be a fallback value anyway

Reply:
I don't understand what do you want to change